src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
```

Comment:
Spaces inside braces please:
```suggestion
import React, { useState } from "react";
```